microraiden/setup.py

```diff
@@ -10,10 +10,84 @@
 with open('requirements-dev.txt') as requirements_dev:
     reqs += requirements_dev.read().split()[2:]
 
+import subprocess
+import distutils
+import os
+from setuptools import Command
+
+DESCRIPTION = 'µRaiden is an off-chain, cheap, scalable and low-latency micropayment solution.'
+
+
+class CompileWebUI(Command):
+    description = 'use npm to compile webui code to raiden/ui/web/dist'
+    user_options = [
+        ('dev', 'D', 'use development preset, instead of production (default)'),
+    ]
+
+    def initialize_options(self):
+        self.dev = None
+
+    def finalize_options(self):
+        pass
+
+    def run(self):
+        npm = distutils.spawn.find_executable('npm')
+        if not npm:
+            self.announce('NPM not found. Skipping webUI compilation', level=distutils.log.WARN)
+            return
+        cwd = os.path.abspath(
+            os.path.join(
+                os.path.dirname(__file__),
+                'microraiden',
+                'webui',
+                'microraiden',
+            )
+        )
+
+        npm_version = subprocess.check_output([npm, '--version'])
+        # require npm 4.x.x or later
+        if not int(npm_version.decode('utf-8').split('.')[0]) >= 4:
+            self.announce(
+                'NPM 4.x or later required. Skipping webUI compilation',
+                level=distutils.log.WARN,
+            )
+            return
+
+        command = [npm, 'install']
+        self.announce('Running %r in %r' % (command, cwd), level=distutils.log.INFO)
+        subprocess.check_call(command, cwd=cwd)
+
+        self.announce('WebUI compiled with success!', level=distutils.log.INFO)
+
+
 config = {
-    'packages': [],
+    'version': '0.0.5',
     'scripts': [],
     'name': 'microraiden',
-    'install_requires': reqs
+    'author': 'Brainbot AG',
+    'author_email': 'info@brainbot.com',
+    'description': DESCRIPTION,
+    'url': 'https://github.com/raiden-network/microraiden/',
+    'include_package_data': True,
+    'license': 'BSD',
+    'keywords': 'raiden ethereum microraiden blockchain',
+    'install_requires': reqs,
+    'packages': find_packages(exclude=['test']),
+    'package_data': {'microraiden': ['data/contracts.json',
+                                     'webui/js/*',
+                                     'webui/index.html',
+                                     'webui/microraiden/dist/umd/microraiden.js']},
+    'classifiers': [
+        'Development Status :: 3 - Alpha',
+        'Intended Audience :: Developers',
+        'License :: OSI Approved :: BSD License',
```

Comment:
I think we should use MIT license here (even though MIT and BSD 2-clause are equivalent)

Reply:
It's what I've copypasted from raiden, maybe it should be changed there too.

Reply:
Yeah, just did that in my branch